dbt/adapters/spark/connections.py

```diff
@@ -273,6 +286,56 @@ def execute(self, sql, bindings=None):
             self._cursor.execute(sql, *bindings)
 
 
+class SessionConnectionWrapper(object):
```

Comment:
What do you think about having this live in a separate file/module, and imported here? I'd hate to confuse users who are trying to debug PyHive/ODBC connection issues that they're experiencing in "real" dbt development/production, and find themselves scanning through this file